src/lib/constants/resourceTypes.ts

```diff
@@ -1,5 +1,8 @@
-export default {
+export const RESOURCE_TYPES = {
   STORY_LISTING: 'node--story_listing',
   STORY: 'node--news_story',
   QA: 'node--q_a',
 } as const
+
+export type ResourceTypeType =
```

Comment:
`ResourceTypeType` and `ListingResourceTypeType` both feel... funky to me. I realize the constant is already SNAKE_CASE_TYPE, but is it too confusing / collide-y to have the types simply be `ResourceType` and `ListingResourceType` ?

Reply:
I had this thought. I'm open to it. The make-everything-make-perfect-sense part of me tells me it should be `...TypeType`, as that is technically "correct", but I can certainly understand why that feels a bit funky.

Reply:
It makes me think that a good rule of thumb might be to not name things `...Type` when working in TypeScript. So we could rename `RESOURCE_TYPES` to `RESOURCE_CATEGORIES` or something along those lines. But that would be a _lot_ of changes at this point.

I don't love `ResourceTypeType` but I think I might prefer it still to `ResourceType`, because I think, in general, it'd be very common to need both `ResourceType` _and_ `ResourceTypeType` (or whatever we'd name it). In other words, I do think this _might_ create a collision at some point, or at least theoretically could when considering the general problem space.

But, again, I'm open to change if people want that. I'm not going to die on this hill. I might not even take a nap on it.